### PR TITLE
Initial page for November CTCFT

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

File: src/meetings/2021-11-22.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+| Info                                           |                          |
+| ---------------------------------------------- | ------------------------ |
+| Time                                           | [See in your time zone]  |
+| Calendar event (with Zoom details)             | [link][cal]              |
+| Social hour calendar event (with Zoom details) | [link][calsh]            |
+| Recording                                      | (link posted after call) |
+
+[see in your time zone]: https://everytimezone.com/s/91c9791f
+[cal]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NnVibTVzcmEwdnZxMTBtYjZ0ZmZxaWtyYXQgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+[calsh]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NGd2N2FzYjBnMHFiNXA4dWdtYWZ2ZmxobmwgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
```
Comment (AngelOnFira) [inline src/meetings/2021-11-22.md:12]: Previously, our calendar links have just been links that allow people to add the event to their calendar, but I don't think it adds them as "attending". This is the link I got from "publishing" the event, which should mean that if someone comes out, they'll be notified next time the event is duplicated. Let me know if this isn't a good way to do it.

File: src/meetings/2021-11-22.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Agenda
+
+The theme for this meeting is Rust in the Linux kernel.
```
Comment (AngelOnFira) [inline src/meetings/2021-11-22.md:16]: @nikomatsakis is there anything else we could add to the "theme" to describe this CTCFT a bit better?

File: src/meetings/2021-11-22.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- (5 min) Opening remarks 👋 ([nikomatsakis])
+- (40 min) Rust in Linux ([ojeda])
+- (5 min) Closing ([nikomatsakis])
+
+[nikomatsakis]: https://github.com/nikomatsakis
+[ojeda]: https://github.com/ojeda
```
Comment (AngelOnFira) [inline src/meetings/2021-11-22.md:23]: @ojeda, I've added the barebones of your section here. Can I get you to write a bit more of a description, and add the Github accounts of Alex/Wedson if they'll be part of the presentation? Feel free to just write in a comment, and I can add it in if that's easier :smiley:

File: src/meetings/2021-11-22.md (hunk 1 above)
Comment (nikomatsakis) [reply src/meetings/2021-11-22.md:12]: I think I always created the links by "publishing", but tbh I don't really understand how google calendar works.

File: src/meetings/2021-11-22.md (hunk 2 above)
Comment (nikomatsakis) [reply src/meetings/2021-11-22.md:16]:
```suggestion
This meeting we've invited some of the people being the work to integrate Rust in the Linux kernel to come and speak. We've asked them to give us a feel for how the integration works and help identify those places where the experience is rough. The expectation is that we can use this feedback as an input when deciding what work to pursue and what features to prioritize for stabilization.
```

Comment (nikomatsakis) [reply src/meetings/2021-11-22.md:16]: Maybe like that?

Comment (technetos) [reply src/meetings/2021-11-22.md:16]: This meeting we've invited some of the people working to integrate Rust into the Linux kernel to come and speak. We've asked them to give us a feel for how the integration works and help identify those places where the experience is rough. The expectation is that we can use this feedback as an input when deciding what work to pursue and what features to prioritize for stabilization.

Comment (AngelOnFira) [reply src/meetings/2021-11-22.md:16]:
```suggestion
This meeting we've invited some of the people working to integrate Rust into the Linux kernel to come and speak. We've asked them to give us a feel for how the integration works and help identify those places where the experience is rough. The expectation is that we can use this feedback as an input when deciding what work to pursue and what features to prioritize for stabilization.
```

File: src/meetings/2021-11-22.md (hunk 3 above)
Comment (ojeda) [reply src/meetings/2021-11-22.md:23]: Of course!

> The Rust for Linux project is adding Rust support to the Linux kernel. While it is still the early days, there are some areas of the Rust language, library and tooling where the Rust project might be able to help out -- for instance, via stabilization of features, suggesting ways to tackle particular problems... This talk will walk through the issues found, along with examples where applicable.

The title should probably be "Rust for Linux" (i.e. the name of the project).

The GitHub are @alex and @wedsonaf.

Comment (AngelOnFira) [reply src/meetings/2021-11-22.md:33]: @ojeda looks good :) got it into the description. I added @alex and @wedsonaf as well, but we can sort who the primary speaker will be later :+1:

**Pull request merged**